### PR TITLE
S-1 to master

### PR DESCRIPTION
**Pull request by tarikkus-lviv**

### CODE REVIEW

**Comment by PetroRavlinko** [comment]
The wrong name for PR, the wrong commit's messages

**Review by PetroRavlinko — CHANGES_REQUESTED** [review]
Change commit messages

File: src/test/java/com/intentsg/shop/repository/OrderRepositoryIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+public class OrderRepositoryIntegrationTest {
+
+    @Autowired
+    private TestEntityManager entityManager;
+
+    @Autowired
+    private OrderRepository orderRepository;
+
+    @Test
+    void whenFindById_thenReturnCart() {
```
Comment (PetroRavlinko) [inline src/test/java/com/intentsg/shop/repository/OrderRepositoryIntegrationTest.java:21]: It's not a cart

File: src/test/java/com/intentsg/shop/services/OrderServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    void testUpdateCart() {
+        Order orderToUpdate = new Order();
+
+        orderService.updateOrder(orderToUpdate, SOME_VALUE);
+
+        verify(orderRepository).save(eq(orderToUpdate));
+    }
+
+    @Test
+    void testGetCart() {
```
Comment (PetroRavlinko) [inline src/test/java/com/intentsg/shop/services/OrderServiceTest.java:52]: I don't see cart

File: src/test/java/com/intentsg/shop/services/OrderServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    private static final Object EXPECTED_CLASS = Order.class;
+
+    @MockBean
+    private OrderRepository orderRepository;
+
+    @Autowired
+    private OrderServiceImpl orderService;
+
+    @Test
+    void testCreateCart() {
```
Comment (vansisto) [inline src/test/java/com/intentsg/shop/services/OrderServiceTest.java:32]: cart -> Order

File: src/test/java/com/intentsg/shop/services/OrderServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        fakeOrder.setId(EXPECTED_ID);
+        given(orderRepository.save(any())).willReturn(fakeOrder);
+
+        Order actualOrder = orderService.createOrder();
+
+        assertEquals(fakeOrder.getId(), actualOrder.getId());
+    }
+
+    @Test
+    void testUpdateCart() {
```
Comment (vansisto) [inline src/test/java/com/intentsg/shop/services/OrderServiceTest.java:43]: Cart -> Order

File: src/test/java/com/intentsg/shop/services/OrderServiceTest.java [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        given(orderRepository.findById(eq(EXPECTED_ID))).willReturn(Optional.of(fakeOrder));
+
+        orderToGet = orderService.getOrder(EXPECTED_ID);
+        verify(orderRepository).findById(eq(EXPECTED_ID));
+
+        assertEquals(EXPECTED_CLASS, orderToGet.getClass());
+    }
+
+    @Test
+    void testDeleteCart() {
```
Comment (vansisto) [inline src/test/java/com/intentsg/shop/services/OrderServiceTest.java:64]: Cart -> Order

File: src/test/java/com/intentsg/shop/repository/OrderRepositoryIntegrationTest.java [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@DataJpaTest
+class OrderRepositoryIntegrationTest {
+
+    @Test
+    void whenFindById_thenReturnOrder() {
```
Comment (PetroRavlinko) [inline src/test/java/com/intentsg/shop/repository/OrderRepositoryIntegrationTest.java:13]: fake test

**Pull request merged**